src/index.js

```diff
@@ -1 +1,47 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import displayScores from './modules/api.js';
+
+// const submitButton = document.querySelector('.submitButton');
```

Comment:
- [ ] I will strongly advise you to remove all commented code in your codebase so as to make your codebase clean and easy to maintain by anyone that will be working on your project 👍🏽 . Kindly work on it  😄